client/src/components/IESupport/IESupport.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactMarkDown from 'react-markdown';
+import content from '@services/content';
+import * as AscLocal from '@components/LocalAsc/LocalAsc';
+import PageTemplate from '@components/PageTemplate/PageTemplate';
+import BreadCrumbs from '@components/BreadCrumbs';
+import ContentBlock from '@components/ContentBlock/ContentBlock';
+
+export interface IProps {}
+
+const IESupport: React.FC<IProps> = () => {
+    return (
+        <PageTemplate>
+            <ContentBlock>
+                <ReactMarkDown
```

Comment:
Dit zou in principe gewoon 1 markdown kunnen zijn toch? In principe lijkt het me mbt beheerbaarheid goed zo min mogelijk structuur af te dwingen.